frame/bags-list/remote-tests/src/migration.rs

```diff
@@ -50,7 +51,7 @@ pub async fn execute<Runtime, Block>(
 
 		use frame_election_provider_support::SortedListProvider;
 		// run the actual migration
-		let moved = <Runtime as pallet_staking::Config>::VoterList::unsafe_regenerate(
```

Comment:
the previous code seems more correct to me. This test is testing the re-generation of `pallet_staking::Config::VoterList`. We don't care if it is backed by what..

Reply:
For this we'll need to move `unsafe_regenerate` to ReadOnly interface, which I guess is far, since it's an unsafe method.